Cython/Compiler/CmdLine.py

```diff
@@ -202,6 +207,7 @@ def get_param(option):
                 sys.stdout.write(usage)
                 sys.exit(0)
             else:
+                sys.stderr.write(usage)
```

Comment:
Not sure if this is desired. It would be helpful for the additional contextual tests I ended up removing, but it's also not a bad thing IMO if the user gives an illegal flag.

Reply:
Seems good to me.